KMaina/Fast-Food-App
# #160219330 Create a Sign In Page

#### What does this PR do?
Allow a user to sign in with their credentials to access the site
#### Description of Task to be completed?
The user should fill in a form with

- An email address

- A password

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-sign-in-page-#160219330`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-sign-in-page-#160219330`

- In the folder where you cloned the repo, you should open the UI folder.

- Open the sign-in.html to see it in your browser for review.

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160219330
